api/rest/workshop_lectors.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import serializers, viewsets
 
-from .lectors import LectorSerializer
 from ..models import WorkshopLector
 
 
 class WorkshopLectorSerializer(serializers.ModelSerializer):
-    lector = LectorSerializer()
```

Comment:
Changed it so that `lectors` and `roles` are requested as a separate resource and combined on the frontend side.